config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -0,0 +1,50 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind-kms
+    decorate: true
+    decoration_config:
+      timeout: 150m
+    always_run: false
+    optional: true # TODO (aramase): make this required and add run_if_changed once the job is passing
```

Comment:
you should add a periodic job too, this job is not likely to run in all presubmits code, we try to have only a reasonable number of jobs in presubmit to have a good balance between code coverage and time to run, 

If you add a periodic job with 12h is enough to catch regressions that breaks your feature

Reply:
We have `optional: true` and `always_run: false` now, so it'll only run when manually triggered. After validating the jobs work as expected:
1. We'll add `run_if_changed` to trigger this job only when encryption at rest files change still keeping it `optional`.
2. We'll add a periodic job to catch regressions.